docs/guides/how_to_guides/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_dbfs.rst

```diff
@@ -0,0 +1,91 @@
+.. _how_to_guides__configuring_metadata_stores__how_to_configure_a_validation_result_store_on_dbfs:
+
+How to configure a Validation Result store on DBFS
+=====================================================
+
+By default, Validation results are stored in the ``uncommitted/validations/`` directory.  Since Validations may include examples of data (which could be sensitive or regulated) they should not be committed to a source control system.  This guide will help you configure a new storage location for Validations on your filesystem.
+
+.. admonition:: Prerequisites: This how-to guide assumes that you have already:
+
+    - Configured a :ref:`Data Context <tutorials__getting_started__initialize_a_data_context>`.
+    - Configured an :ref:`Expectation Suite <tutorials__getting_started__create_your_first_expectations>`.
+    - Configured a :ref:`Checkpoint <tutorials__getting_started__set_up_your_first_checkpoint>`.
+    - Determined a new storage location where you would like to store Validations. This can either be a local path, or a path to a secure network filesystem.
+
+Steps
+-----
+
+1. **Configure a new folder on your filesystem where Validation results will be stored.**
+
+    Create a new folder where you would like to store your Validation results, and move your existing Validation results over to the new location. In our case, the name of the Validation result is ``npi_validations`` and the path to our new storage location is ``shared_validations/``.
+
+    .. code-block:: bash
+
+        # in the great_expectations/ folder
+        mkdir shared_validations
+        mv uncommitted/validations/npi_validations/ uncommitted/shared_validations/
+
+
+2. **Identify your Data Context Validations Store**
```

Comment:
I'd remove this step since you're basically re-explaining the concept of validation stores here. You could just say "find the stores section and add a new store as follows:"

Reply:
file is not being included in PR anymore.